src/index.ts

```diff
@@ -9,6 +9,7 @@
  * @since 1.0.0
  */
 
+import type { Environment } from 'genesys-cloud-service-discovery-web';
```

Comment:
oh, was wondering how either of these files were resolving the type without the import since it's used in the file.  Now, I see that it's defined in globals.d.ts.

I was worried that VS Code's `organize imports` task would remove the import in favor of the global, but I just tried it and it looks like the import still sticks around.